test/ecto/repo/autogenerate_test.exs

```diff
@@ -8,6 +8,7 @@ defmodule Ecto.Repo.AutogenerateTest do
 
     @timestamps_opts [inserted_at: :created_on]
     schema "manager" do
+      belongs_to :company, Company
```

Comment:
btw, looking at docs for [`has_one`](https://github.com/elixir-ecto/ecto/blob/v2.1.1/lib/ecto/schema.ex#L634:L635):

> The other schema often has a `belongs_to` field with the reverse association.

as in this case, if the other schema does not have the belongs_to, trying to `Changeset.change` FK will fail. Perhaps it's obvious but maybe additional docs would be nice (not sure how they'd look like though)